src/components/index.d.ts

```diff
@@ -185,6 +185,34 @@ interface downstreamButtonProps {
   ariaLabelId?: string
 }
 
+export declare const Modal: {
+  ({
+    ariaLabelledBy,
+    ariaDescribedBy,
+    children,
+    isOpen,
+    onDismiss,
+  }: {
+    /** Id of an element that labels the modal */
+    ariaLabelledBy?: string
```

Comment:
I was always confused about this spelling. Last I checked, `labeled` has only one `l`, but it seems like this is the standard haha